src/content/docs/getting-started/install.md

```diff
@@ -2,34 +2,31 @@
 title: Installation
 ---
 
-## Prerequisites
+## Membrane Web IDE
 
-- Git -- [How to install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- Vscode -- [Download and install VSCode](https://code.visualstudio.com/download)
+Membrane's primary interface is the Membrane Web IDE (fork of VS Code). The IDE is where you deploy Membrane programs, set up cron jobs, manually run programs, inspect logs, and more.
 
-## VSCode Extension
+Visit [membrane.io/ide](https://www.membrane.io/ide) to sign up and open your Membrane workspace.
 
-Membrane's primary interface is its VSCode extension. It handles deploying programs, listing which programs are running, displaying logs, and more.
+<!-- TODO: add gif of web IDE startup -->
 
-Install the extension via [VSCode's marketplace website](https://marketplace.visualstudio.com/items?itemName=membrane.membrane) or by searching `Membrane` in the extensions market place directly in vscode.
+### Welcome screen
 
-<video src="/cloud-assets/membrane-install.mp4" muted autoplay loop></video>
+When the IDE starts up you'll see Membrane's Program Explorer and Program Logs panels and a welcome screen. The welcome screen provides a tour of the Explorer and Logs.
 
-### Welcome Screen
-
-When the extension starts it displays a welcome screen. This screen aids in signing in, opening Membrane's [Program Explorer]() and [Program Logs]() panels, and giving a deeper tour of features. It also links off to an interactive tutorial program that comes pre-installed in every user's workspace.
```

Comment:
Were `Program Explorer` and `Program Logs` meant to link to to-be-written sections?

Reply:
I had originally intended to write a explainer for those panels. I'm not sure it's entirely necessary though.